engine/src/main/java/org/terasology/rendering/cameras/ViewFrustum.java

```diff
@@ -186,6 +240,9 @@ public boolean intersects(AABB aabb) {
 
     /**
      * Returns true if the given sphere intersects the given AABB.
+     *
+     * @deprecated This is scheduled for removal in an upcoming version method will be replaced with JOML implementation
+     *     {@link #intersects(Vector3fc, float)}.
      */
     public boolean intersects(Vector3f position, float radius) {
         for (int i = 0; i < 6; i++) {
```

Comment:
The deprecated method should redirect to the new one to get rid of duplication:
`return intersects(JomlUtil.from(position), radius);`